src/include/cluster/dialogs.rb

```diff
@@ -1293,14 +1304,23 @@ def csync2_turn_on
       y2debug("Start and enable csync2.socket.")
 
       tcp_ports = []
```

Comment:
Could remove the predefine of "tcp_ports" like your previous change.

Reply:
sure, I could even clean it a little if you want, it is an almost complete example, so I could finish it completely or you could just create a new branch from it